### ISSUE TITLE
Supress sensitive information in terminal output during make all

### ISSUE THREAD

**Issue by emkll** [freedomofpress/securedrop-workstation#417]
The provisioning process will write sensitive information to standard output [1] . We should suppress the output of that (perhaps `quiet: true` would work here [2]?

[1] : https://github.com/freedomofpress/securedrop-workstation/blob/master/dom0/sd-whonix-hidserv-key.sls
[2]: https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.file.html

**Comment by conorsch** [comment]
> perhaps quiet: true would work here [2]?

Not quite; [from the docs](https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.file.html): 

>**quiet** 
 Do not raise any exceptions. E.g. ignore the fact that the file that is tried to be edited does not exist and nothing really happened.

What we really want is the Salt equivalent of [Ansible's `no_log`](https://docs.ansible.com/ansible/latest/reference_appendices/logging.html#protecting-sensitive-data-with-no-log), assuming such a beast exists. After some quick searching, it looks like `cmd.run` has a [`hide_output` option](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.cmd.html), but nothing similar for `file`.

**Comment by zenmonkeykstop** [comment]
Keeping for now - in future we may have better options for this during initial setup, but for now we still need to suppress said info.
